.github/integration/sda/aai-mock/userinfos/test-user.yaml

```
@@ -2,7 +2,7 @@ web-display-name: "Test User - admin"
 
 sub: "testu@lifescience-ri.eu"
 
-name: "Test User"
+name: "testu@lifescience-ri.eu"
```

```suggestion
name: "Test User"
```
This should be as it is.

---

reverted the commit ea0908da